src/imitation/algorithms/preference_comparisons.py

```diff
@@ -760,6 +792,10 @@ def __init__(
                 creating fragments. Since fragments are sampled with replacement,
                 this is usually chosen > 1 to avoid having the same transition
                 in too many fragments.
+            initial_comparison_frac: fraction of the total_comparisons argument
+                to train() that will be sampled before the rest of training begins
+                (using the randomly initialized agent). This can be used to pretrain
+                the reward model before the agent is trained on the learned reward.
```

Comment:
```suggestion
                the reward model before the agent is trained on the learned reward,
                to help avoid irreversibly learning a bad policy from an untrained reward.
```
Paraphrasing the justification given in appendix A of https://arxiv.org/pdf/1706.03741.pdf